worlds/shivers/Items.py

```diff
@@ -1,132 +1,192 @@
+        SHIVERS_ITEM_ID_OFFSET + 94, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Wax Always Available in Shaman Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 95, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Ash Always Available in Office": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 96, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Ash Always Available in Burial Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 97, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Oil Always Available in Prehistoric Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 98, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Cloth Always Available in Egypt": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 99, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Cloth Always Available in Burial Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 100, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Wood Always Available in Workshop": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 101, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Wood Always Available in Blue Maze": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 102, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Wood Always Available in Pegasus Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 103, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Wood Always Available in Gods Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 104, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Crystal Always Available in Lobby": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 105, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Crystal Always Available in Ocean": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 106, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Sand Always Available in Greenhouse": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 107, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Sand Always Available in Ocean": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 108, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Metal Always Available in Projector Room": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 109, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Metal Always Available in Bedroom": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 110, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Metal Always Available in Prehistoric": ItemData(
+        SHIVERS_ITEM_ID_OFFSET + 111, ItemType.IXUPI_AVAILABILITY, ItemClassification.filler
+    ),
+    "Heal": ItemData(SHIVERS_ITEM_ID_OFFSET + 112, ItemType.FILLER, ItemClassification.filler),
 
+    # Victory item
+    f"Mt. Pleasant Tribune: {Constants.years_since_sep_30_1980} year Old Mystery Solved!": ItemData(
```

Comment:
Wait a minute, no.

Your location_name_to_id cannot change from one day to the next like this.
It is static "per run of AP", yes, but that is not enough. It has to be static across a *frozen build* of AP, no matter which day you run that build on.

The website loads location_name_to_id at different points of time / on different schedules for different purposes. Some things only happen once per **update** (If we didn't have the server crashes, options pages would still use the datapackage from **the release date of 0.5.0**), whereas for *generation*, a new instance of AP is loaded.

If you really want this, you'll have to make different items per year somehow. E.g. reserve like 100 of them, ahead of time, thinking either AP will be dead by then, or we'll have a more elegant solution to this datapackage issue.

Reply:
I have been informed this may be an event item. If it is code = None, then ignore everything I said.

Reply:
The item does appear to have a code in its ItemData, which makes me assume it's a real item

Reply:
Ah shit really? And here I was thinking I was doing something fun. What would that actually affect / break (In the window of being around Sep 30th each year)? I don't really know how everything works.

Hmm like I know it affects the web host in this way, though ugh I wish some items could be explicitly removed from these lists (since plenty of games have items that shouldn't be plando but aren't events).
<img width="542" alt="Screenshot 2024-10-18 at 22 34 14" src="https://github.com/user-attachments/assets/9bd9b298-b3cb-416c-974b-545e86db654a">

Because I could generate a bunch of these items with static years and the generation picks the one it wants to use at run time. (I mean could that really just be restricted to a couple years, since if it's too old it just falls off the list and too new then it won't matter. i.e. current +-1 year 43 44 and 45, then when next year rolls around it's 44 45 and 46?)

But again the webhost showing those is annoying.

And I know items show up in the tracker if received, but my assumption is that comes from whatever the generation said?

Reply:
@NewSoupVi didn't see the other messages, this is not an even item, it could be but the point was to have it show up when the player goals. So in any case it's never an item that will be randomized

Reply:
You can override options like LocalItems and provide your own set of `valid_keys`.

(In fact, if you removed this item from all `valid_keys` of all generic options, I could see an argument that *maybe* it's fine to have an item that changes names? I'm not sure though 🥴 I'd feel much more comfortable if you just reserved 100 different items or so ahead of time)

Reply:
@NewSoupVi Ah, I'll have to look into doing that, because we wouldn't want them in those options. What about the rolling +-1 item buffer since this is an item that only changes yearly we wouldn't really ever need more than 3 at a time.

Reply:
Hmm... a rolling item buffer *might* be okay. The reason I'm a bit wary is because it would still mean that some parts of the website would be working with:

```
43 years ago: ####0001
44 years ago: ####0002
45 years ago: ####0003
```

where other parts would be working with

```
44 years ago: ####0001
45 years ago: ####0002
46 years ago: ####0003
```

Because yes, this would mean that any name that would actually be used is always in there as long as there is a server restart at least once a year (lol), but which name belongs to which ID would be mismatched.
I don't *think* that causes any issues, but I'm not sure. I'd have to verify with other core devs

Reply:
I can associate the id with the year 

```
43 years ago: ####00043
44 years ago: ####00044
45 years ago: ####00045
```

```
44 years ago: ####00044
45 years ago: ####00045
46 years ago: ####00046
```

Reply:
Oh sure. At that point I don't really see the benefit over just reserving a bunch ahead of time, other than I guess "it works forever", which I don't imagine will be relevant but it is *technically* slightly safer. Lol

Reply:
I'm both lazy and don't want a bunch of extra stuff laying around that won't get used lol.

Reply:
@NewSoupVi ok not that I had time to do this, done. Have my rolling 3 item buffer and removed them from valid keys so they don't show on web.